web_timeline/static/src/css/web_timeline.css

```diff
@@ -9,7 +9,7 @@
     background: #DCDCDC;
 }
 
-.vis-item .vis-item-overflow {
+.vis-item .vis-item-overflow:hover {
```

Comment:
Why this change?

Reply:
There is too much text overflow in the timeline view. This allows to keep the text inside the "element" of the timeline view, except when you put the mouse over. This is a usability improvement.